doc/api/crypto.md

```diff
@@ -1348,27 +1348,29 @@ keys.
 ### `keyObject.export([options])`
 <!-- YAML
 added: v11.6.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/37081
+    description: Added support for `'jwk' format`.
 -->
 
 * `options`: {Object}
-* Returns: {string | Buffer}
+* Returns: {string | Buffer | JWK}
```

Comment:
Shouldn't this be `Object` instead? `JWK` isn't a type

Reply:
I thought about that too, one way or the other is fine with me.

Reply:
Yeah I was stewing over this also. It should likely be Object but it would be good to have a reference section in the crypto.md that discusses the JWK structure and where to go for more information.

Reply:
✅